content/guides/tooling/sdks/sdk-versioning.md

```diff
@@ -13,63 +13,90 @@ alias_paths: []
+  </Tile>
+  <Tile type="branch" title="3. New Application" href="/guides/tooling/sdks/sdk-versioning/#box-core-sdk-versions-and-artifacts-overview">
+    If you are creating a new application:
 
-1. If you are creating a new application, use `v10` of the Box core SDK package.
-2. If you have an existing application relying on the **Box Next Generation SDK** and you wish to further develop your project, replace the name of this library in the package manager with the Box core SDK package (`v10.0.0`). Object imports for most SDKs will persist the same and your code will work as-is; the TypeScript SDK requires additional steps for migrating. Check the section below for detailed guides.
-3. If you have an existing application relying on **Box core SDK** and you wish to further develop your project, once released, bump the library version by one major release. See the table above for a detailed breakdown of recommended versions.
-4. If you have an existing application that you don’t plan to change, ensure your package manager includes the version of SDK you are using to prevent an accidental rebuild that pulls in a version you aren’t expecting.
+    **Action** Use ≥`v10.0.0` of the Box core SDK package.
+  </Tile>
+  <Tile type="info" title="4. No Active Development" href="/guides/tooling/sdks/sdk-versioning/#box-core-sdk-versions-and-artifacts-overview">
+    If you have an existing application that you don't plan to change:
+
+    **Action** Ensure your package manager includes the specific version of the SDK version to prevent accidental updates. We strongly recommend upgrading to the major version which includes consolidated packages to receive ongoing security patches and improvements.
+  </Tile>
+</TileGrid>
 
 ## Versioning overview
 
-### Box SDK versions and artifacts
+### Box core SDK versions and artifacts overview
+
+| Repository name  | Artifact name |  Includes both packages | Includes the only generated package|
+|--------------|------|---------|----------|
+| [`box-python-sdk`][python-repo] | `boxsdk` | [`v4.X.Y`][python-combined] | [`≥v10.0.0`][python-v10] |
+| [`box-node-sdk`][node-repo]  | `box-node-sdk` | [`v4.X.Y`][node-combined] | [`≥v10.0.0`][node-v10] |
+| [`box-java-sdk`][java-repo] |  `box-java-sdk` | [`v5.X.Y`][java-combined] | [`≥v10.0.0`][java-v10] |
+| [`box-windows-sdk-v2`][windows-repo] | `Box.V2`, `Box.V2.Core` | [`v6.X.Y`][windows-combined] | [`≥v10.0.0`][windows-v10] |
+| [`box-ios-sdk`][ios-repo] | `BoxSDK` | [`v6.X.Y`][ios-combined] | [`≥v10.0.0`][ios-v10] |
+
+## Box Next Gen SDKs deprecation
+
+As of September 17, 2025 Box Next Generation SDKs are no longer supported as separate artifacts. Don’t worry, your existing code will continue to work without changes. You can still use your applications based on Box Next Generation SDKs with no impact, but you won't receive new features, updates, or bug fixes.
```

Comment:
I think some time ago we changed this paragraph a bit, as one of the customers was asking about security updates.

https://github.com/box/box-typescript-sdk-gen?tab=readme-ov-file#deprecation-notice

This part was added

> We will be still providing patches for security vulnerabilities until version 4 (containing both box-node-sdk and box-typescript-sdk-gen packages) of [box-node-sdk](https://github.com/box/box-node-sdk) is released.

Should we add similar sentence there? It should be obsolete when combined-sdks are released though.